vite.config.ts

```diff
@@ -1,7 +1,8 @@
 import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react'
+import eslint from 'vite-plugin-eslint'
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [react()]
+  plugins: [react(), eslint()]
```

Comment:
Please use conditional config to distinguish between Dev and Prod. May it is not a good idea to execute eslint plugin during a production build.
https://vitejs.dev/config/#conditional-config

Reply:
Done